examples/chef/devices/rootnode_airqualitysensor_e63187f6c9.matter

```diff
@@ -0,0 +1,2165 @@
+    ram      attribute featureMap default = 63;
+    ram      attribute clusterRevision default = 3;
+  }
+
+  server cluster Pm25ConcentrationMeasurement {
+    callback attribute measuredValue default = 42;
+    callback attribute minMeasuredValue default = 1;
+    callback attribute maxMeasuredValue default = 400;
+    callback attribute peakMeasuredValue default = 90;
+    callback attribute peakMeasuredValueWindow default = 3600;
+    callback attribute averageMeasuredValue default = 35;
+    callback attribute averageMeasuredValueWindow default = 3600;
+    callback attribute uncertainty default = 4;
+    callback attribute measurementUnit default = 4;
+    callback attribute measurementMedium default = 0;
+    callback attribute levelValue default = 2;
+    callback attribute generatedCommandList;
+    callback attribute acceptedCommandList;
+    callback attribute eventList;
+    callback attribute attributeList;
+    ram      attribute featureMap default = 63;
+    ram      attribute clusterRevision default = 3;
+  }
+
+  server cluster FormaldehydeConcentrationMeasurement {
+    callback attribute measuredValue default = 10;
+    callback attribute minMeasuredValue default = 0;
+    callback attribute maxMeasuredValue default = 200;
+    callback attribute peakMeasuredValue default = 10;
+    callback attribute peakMeasuredValueWindow default = 7200;
+    callback attribute averageMeasuredValue default = 2;
+    callback attribute averageMeasuredValueWindow default = 7200;
+    callback attribute uncertainty default = 0;
+    callback attribute measurementUnit default = 3;
+    callback attribute measurementMedium default = 0;
+    callback attribute levelValue default = 2;
+    callback attribute generatedCommandList;
+    callback attribute acceptedCommandList;
+    callback attribute eventList;
+    callback attribute attributeList;
+    ram      attribute featureMap default = 63;
+    ram      attribute clusterRevision default = 3;
+  }
+
+  server cluster Pm1ConcentrationMeasurement {
+    callback attribute measuredValue default = 39;
+    callback attribute minMeasuredValue default = 1;
+    callback attribute maxMeasuredValue default = 400;
+    callback attribute peakMeasuredValue default = 70;
+    callback attribute peakMeasuredValueWindow default = 3600;
+    callback attribute averageMeasuredValue default = 41;
+    callback attribute averageMeasuredValueWindow default = 3600;
+    callback attribute uncertainty default = 4;
+    callback attribute measurementUnit default = 4;
+    callback attribute measurementMedium default = 0;
+    callback attribute levelValue default = 1;
+    callback attribute generatedCommandList;
+    callback attribute acceptedCommandList;
+    callback attribute eventList;
+    callback attribute attributeList;
+    ram      attribute featureMap default = 63;
+    ram      attribute clusterRevision default = 3;
+  }
+
+  server cluster Pm10ConcentrationMeasurement {
```

Comment:
I assume "PM2.5 Concentration Measurement" is disabled here on purpose?

Reply:
No. The [PM2.5](https://github.com/project-chip/connectedhomeip/pull/29436/files#diff-8b1b09c1c7162c3750a2c9ea83273564c3b54bb87f1a803410d19e5e256c08a2R2015-R2033) which is generated in front of PM1 which follows the ZAP file's order